### PR TITLE
Use RuntimeEnvironment to determine current architecture in tests

### PR DESCRIPTION
**Pull request by ChengTian**
Making the code DRYer.

@victorhurdugaci  @BrennanConroy

### CODE REVIEW

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs [hunk 1]
```diff
@@ -9,11 +9,18 @@
 using System.Security.Cryptography;
 using System.Text;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
```
Comment (BrennanConroy) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: .Abstractions now

**Comment by BrennanConroy** [comment]
Other than comment, :shipit:

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs [hunk 2]
```diff
@@ -305,17 +312,9 @@ public static string ComputeSHA(string path)
             }
         }
 
-        public static string GetCurrentRuntimeArchitecture()
+        public static IRuntimeEnvironment GetCurrentRuntimeEnvironment()
```
Comment (victorhurdugaci) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Make it a property?

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs (hunk 1 above)
Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Did you forget to do that part during renaming?
https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.Runtime.Abstractions/CallContextServiceLocator.cs#L4

Comment (BrennanConroy) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Sorry, I misread, thought you put .Interface

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs (hunk 2 above)
Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Fixed

File: test/Microsoft.Framework.CommonTestUtils/TestUtils.cs [hunk 3]
```diff
@@ -9,11 +9,15 @@
 using System.Security.Cryptography;
 using System.Text;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
 
 namespace Microsoft.Framework.CommonTestUtils
 {
     public static class TestUtils
     {
+        public static IRuntimeEnvironment CurrentRuntimeEnvironment = CallContextServiceLocator.Locator.ServiceProvider
```
Comment (victorhurdugaci) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Um... this is not right. The static field can be initialized at any time, maybe even before the service provider

Comment (victorhurdugaci) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Create a static property that gets the value the first time it is invoked

Comment (ChengTian) [inline test/Microsoft.Framework.CommonTestUtils/TestUtils.cs]: Fixed

**Comment by davidfowl** [comment]
:shipit:

**Pull request merged**